jscomp/main/js_main.ml

```diff
@@ -169,6 +169,12 @@ let buckle_script_flags : (string * Arg.spec * string) list =
    " (experimental) Set the string to be evaluated, note this flag will be conflicted with -bs-main"
   )
   ::
+  (
+    "-bs-cmi-only",
+    Arg.Set Js_config.cmi_only,
+    " Stopped after generating cmi file"
```

Comment:
In keeping with the documentation for other flags, this should probably be present tense? E.g. `Stop after generating cmi file'

Reply:
thanks